storybook/storybook-react/src/Card/Card.docs.mdx

```diff
@@ -0,0 +1,21 @@
+import { Canvas, Controls, Markdown, Meta, Primary } from "@storybook/blocks";
+import * as CardStories from "./Card.stories.tsx";
+import README from "../../../../packages/css/src/card/README.md?raw";
+
+<Meta of={CardStories} />
+
+<Markdown>{README}</Markdown>
+
+<Primary />
+
+<Controls />
+
+## Card with tagline
+
+Een card kan een _tagline_ hebben, tekst voor de titel.
+Om de volgorde voor niet-visuele _user agents_ (zoals een schermlezer) logisch te houden,
```

Comment:
Is it too imprecise to just mention screen readers everywhere? “niet-visuele _user agents_” feels a bit too technical.

```suggestion
Om de volgorde voor schermlezers logisch te houden,
```

Reply:
I've chosen these words because I wanted to include braille readers and any non-visual user agent I don't know about :D 

We could do something like this, to make it clear we don't consider screen readers the only non-visual UA:

```suggestion
Om de volgorde voor bijvoorbeeld schermlezers logisch te houden,
```

But if that's too nitpicky, we can go with your version.

Reply:
If you’re naming it a ‘braille reader’, that’s a member of the ‘screen reader’ group as well, right? :)

Reply:
Ah, you're right! I thought a screen reader is just a voice reader, but braille readers are part of that group as well. Shorter is better.